### DISCUSSION TITLE
to_dict is not defined and sourcePaths is not defined error

### DISCUSSION THREAD

**Discussion by KPacheco-TailorCare** [AltimateAI/vscode-dbt-power-user#858] [Q&A]
I have installed dbt Power User but when selecting a query to run and view in the query results I get : 

An error occured while trying to execute your query: name 'to_dict' is not defined

I also see a runtime status error:

sourcePaths is not defined in project in /home/kpacheco/git/tc-dp-tailorcare-dbt

Nothing seems to really be working as far as the lineage etc

**Comment by anandgupta42** [comment]
@KPacheco-TailorCare 
Couple of questions:
1. What do you see in the problems tab? 
2. What's the version of the extension installed on your machine?
It seems the underlying Python environment is not right. Have you gone through the extension setup wizard https://docs.myaltimate.com/setup/installation/ ?

**Comment by KPacheco-TailorCare** [comment]
Hi thank you for your reply,

dbt Power User version is: v0.30.1
dbt version is: 1.7.5

I do notice this error all of a sudden regarding my dbt_project.yml file. I
am though able to connect to my db and run all dbt commands

[image: image.png]

On Thu, Jan 25, 2024 at 7:11 PM anandgupta42 ***@***.***>
wrote:

> @KPacheco-TailorCare <https://github.com/KPacheco-TailorCare>
> Couple of questions:
>
>    1. What do you see in the problems tab?
>    2. What's the version of the extension installed on your machine?
>    It seems the underlying Python environment is not right. Have you gone
>    through the extension setup wizard
>    https://docs.myaltimate.com/setup/installation/ ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/AltimateAI/vscode-dbt-power-user/discussions/858#discussioncomment-8251200>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDZVH7ILTSXRVAFKFT3OCFTYQLYCNAVCNFSM6AAAAABCLFU6OSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DENJRGIYDA>
> .
> You are receiving this because you were mentioned.Message ID:
> <AltimateAI/vscode-dbt-power-user/repo-discussions/858/comments/8251200@
> github.com>
>


-- 

  Katherine Pacheco

  Senior Data Analyst

C: 305-731-6674

E: ***@***.***
